### ISSUE TITLE
strategy/rl_agent.py

### ISSUE THREAD

**Issue by temur3559** [temur3559/UltimateCryptoBotV2#23]
import torch
import torch.nn as nn
import torch.optim as optim
import numpy as np

class PPOPolicy(nn.Module):
    def __init__(self, state_dim, action_dim):
        super(PPOPolicy, self).__init__()
        self.fc = nn.Sequential(
            nn.Linear(state_dim, 128),
            nn.ReLU(),
            nn.Linear(128, action_dim),
            nn.Softmax(dim=-1)
        )

    def forward(self, state):
        return self.fc(state)

class PPOAgent:
    def __init__(self, state_dim, action_dim, lr=1e-4):
        self.policy = PPOPolicy(state_dim, action_dim)
        self.optimizer = optim.Adam(self.policy.parameters(), lr=lr)

    def select_action(self, state):
        state = torch.FloatTensor(state).unsqueeze(0)
        probs = self.policy(state)
        action = torch.multinomial(probs, num_samples=1)
        return action.item()

    def train(self, states, actions, rewards):  # Oddiy update, PPO uchun to‘liq optimizatsiya talab qilinadi
        pass  # Bu joyga keyin to‘liq PPO optimizatsiyasi yoziladi
